src/api/json/catalog.json

```diff
@@ -5565,8 +5565,13 @@
     {
       "name": "TunnelHub",
       "description": "TunnelHub Automation Configuration File",
-      "fileMatch": ["tunnelhub.yml", "tunnelhub.yaml"],
       "url": "https://json.schemastore.org/tunnelhub.json"
```

Comment:
Why did you remove the `fileMatch` for TunnelHub?

Reply:
Oops! Was just a C&P error. Undone! sorry!